kasa/__init__.py

```diff
@@ -58,6 +58,8 @@
     "DeviceType",
     "Feature",
     "FeatureType",
+    "FeatureNames",
+    "StandardFeature",
```

Comment:
I haven't thought this through nor experimented with it, but I'm wondering if we should just define the standard actions inside `Feature`? Basically leaving us with an API like:
```
await dev.features[Feature.Brightness].set_brightness(123)
await dev.features[Feature.State].turn_on()
```
or even
```
from kasa import Color
await dev.features[Color].set_hsv(1, 1, 1)
```
where `Color` is the name of the interface that a feature implements?